homeassistant/components/timer/__init__.py

```diff
@@ -60,6 +60,8 @@
 SERVICE_PAUSE = "pause"
 SERVICE_CANCEL = "cancel"
 SERVICE_CHANGE = "change"
+SERVICE_ATLEAST = "atleast"
+SERVICE_ATMOST = "atmost"
```

Comment:
I think to match the style of the rest of the home assistant services it should be 

```python
SERVICE_ATLEAST = "at_least"
SERVICE_ATMOST = "at_most"
```

Example from fan: https://github.com/home-assistant/core/blob/4e8186491cf655c8e61c6cf0e955e89d89ce916a/homeassistant/components/fan/__init__.py#L56-L57